debian/trusty/foreman/compat

```diff
@@ -1 +1 @@
-7
```

Comment:
Maybe all changes to trusty files should be reverted so we don't imply they do work?